.babelrc

```diff
@@ -1,11 +1,13 @@
 {
-  "presets": ["react"],
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
   "env": {
     "test": {
       "plugins": [
-        "transform-es2015-modules-commonjs",
-        "transform-es2015-classes",
-        "transform-object-rest-spread"
+        "transform-modules-commonjs",
+        "transform-classes"
```

Comment:
Not sure if we need that anymore. Maybe remove it and readd it only when writing more tests and it turns out we need it?

Reply:
Yep